go.mod

```diff
@@ -7,8 +7,9 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20210719174558-55d730c80aa9
+	github.com/openshift/api v0.0.0-20210722202039-91e26cfb36fc
```

Comment:
The version bump presumably came from importing github.com/openshift/cluster-ingress-operator, and now that that import is gone, the version bump should be unnecessary.  We can keep it if you prefer, but it is a little odd and adds some unnecessary changes to the PR.

Reply:
Ok, removed it.